airbyte-integrations/connectors/source-file/integration_tests/file_formats_test.py

```diff
@@ -40,19 +40,21 @@ def check_read(config, expected_columns=10, expected_rows=42):
 
 
 @pytest.mark.parametrize(
-    "file_format, extension, expected_columns, expected_rows",
+    "file_format, extension, expected_columns, expected_rows, filename",
     [
-        ("csv", "csv", 8, 5000),
-        ("json", "json", 2, 1),
-        ("excel", "xls", 8, 50),
-        ("excel", "xlsx", 8, 50),
-        ("feather", "feather", 9, 3),
-        ("parquet", "parquet", 9, 3),
+        ("csv", "csv", 8, 5000, "demo"),
+        ("json", "json", 2, 1, "demo"),
+        ("ndjson", "ndjson", 2, 10, "ndjson_nested"),
```

Comment:
in `destination-local-json` we are using `.jsonl` extensions, is `ndjson` standard way of referring to it? Should we harmonize in sources & destinations?